Analysis/Tasks/PWGHF/HFCandidateCreator3Prong.cxx

```diff
@@ -154,11 +154,23 @@ struct HFCandidateCreator3ProngMC {
   {
     int8_t sign = 0;
     int8_t result = N3ProngDecays;
+    int8_t DecayChannel = 0;
+    int8_t size;
+    std::vector<int> arrDaughIndex;
+    std::vector<int> arrPDGDaugh;
+    std::vector<int> arrPDGResonant1 = {2212, 313};
+    std::vector<int> arrPDGResonant2 = {2224, 321};
+    std::vector<int> arrPDGResonant3 = {3124, 211};
```

Comment:
Are vectors needed here? It seems, their lengths are constant and therefore could be arrays instead.

Reply:
Good point. Thanks. Indeed is better to use the arrays.